book/src/web-client/mint_consume_send_tutorial.md

```diff
@@ -0,0 +1,226 @@
+* Consuming notes to fund an account
+* Sending tokens to other users
+
+## Step 1: Minting tokens from the faucet
+To mint notes with tokens from the faucet we created, Alice can use the web client's `new_mint_transaction()` function.
+
+Below is an example of a transaction request minting tokens from the faucet for Alice. 
+
+Add this snippet to the `src/webClient.ts` function that we created in the previous chapter:
+```ts
+const targetId = AccountId.from_hex(account.id().to_string());
+const faucetId = AccountId.from_hex(faucet.id().to_string());
+const amount = BigInt(1000);
+
+await client.fetch_and_cache_account_auth_by_pub_key(faucetId);
+await client.sync_state();
+
+await client.new_mint_transaction(
+  targetId,          // target wallet id
+  faucetId,          // faucet id
+  NoteType.public(), // Note Type
+  amount             // use the passed amount
+);
+
+await client.sync_state();
+```
+
+## Step 2: Identifying consumable notes
+Once Alice has minted a note from the faucet, she will eventually want to spend the tokens that she received in the note created by the mint transaction. 
+
+Minting a note from a faucet on Miden means a faucet account creates a new note targeted to the requesting account. The requesting account must consume this note for the assets to appear in their account.
+
+To identify notes that are ready to consume, the Miden web client has a useful function `get_consumable_notes`. It is also important to sync the state of the client before calling the `get_consumable_notes` function. 
+
+*Tip: If you know the expected number of notes after a transaction, use `await` or a loop condition to verify their availability before calling `get_consumable_notes`. This prevents unnecessary application idling.*
+
+#### Identifying which notes are available:
+
+```ts
+consumable_notes = await client.get_consumable_notes(accountId);
+```
+
+## Step 3: Consuming multiple notes in a single transaction:
+Now that we know how to identify notes ready to consume, let's consume the notes created by the faucet in a single transaction. After consuming the notes, Alice's wallet balance will be updated.
+
+The following code snippet identifies and consumes notes in a single transaction.
+
+Add this snippet to the `src/webClient.ts` function:
+```ts
+// get available notes
+let data = await client.get_consumable_notes(aliceAccountId);
+let noteIds = data.map(item => item.input_note_record().id().to_string());
+
+// consume notes
+console.log('consuming notes...');
+await client.fetch_and_cache_account_auth_by_pub_key(aliceAccountId);
+await client.new_consume_transaction(
+  aliceAccountId, // account id 
+  noteIds         // array of note ids to consume
+    );
+```
+
+## Step 4: Sending tokens to other accounts
+After consuming the notes, Alice has tokens in her wallet. Now, she wants to send tokens to her friends. She has two options: create a separate transaction for each transfer or batch multiple notes in a single transaction.
```

Comment:
> After consuming the notes, Alice has tokens in her wallet.

Can we show that with a `console.log()`? Not sure if we can show the current account vault state